src/app/forms/components/defect-select/defect-select.component.html

```diff
@@ -32,7 +34,14 @@ <h1 class="govuk-heading-l">Select a defect</h1>
   <div *ngIf="selectedItem" class="list_container">
     <ol class="list">
       <li *ngFor="let deficiency of selectedItem?.deficiencies">
-        <a class="browse__normal-link govuk-link--no-visited-state" (click)="handleSelect(deficiency, types.Deficiency)"
+        <a
+          class="browse__normal-link govuk-link--no-visited-state"
+          [id]="
+            deficiency.deficiencySubId
+              ? 'selectedDefect-' + deficiency.deficiencyId + deficiency.deficiencySubId
+              : 'selectedDefect-' + deficiency.deficiencyId
+          "
```

Comment:
```suggestion
          [id]="'selectedDefect-' + deficiency.deficiencyId + deficiency.deficiencySubid ? deficiency.deficiencySubId : ''"
```
just to shorten a little